src/Tribe/Views/V2/View.php

```diff
@@ -806,12 +819,18 @@ public function prev_url( $canonical = false, array $passthru_vars = [] ) {
 		}
 
 		// Make sure the view slug is always set to correctly match rewrites.
-		$url = add_query_arg( [ 'eventDisplay' => $this->slug ], $url );
+		$query_args['eventDisplay'] = $this->slug;
 
-		$url = $prev_page->count() > 0 ? add_query_arg( $query_args, $url ) : '';
+		if ( $prev_page->count() > 0 ) {
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 96fd42b34267d0380436f3640d578e65</sub>